README.md

```diff
@@ -30,7 +34,7 @@ with any keyboard, including your laptop’s.
 Main Benefits
 --------------------------------------------------------------------------------
 
-- <kbd>Shift</kbd>, <kbd>Backspace</kbd>, <kbd>Return</kbd> under the thumbs!
+- <kbd>Alt</kbd>/<kbd>Shift</kbd>, <kbd>Backspace</kbd>, <kbd>Return</kbd> under the thumbs!
```

Comment:
`Alt` is already under the thumbs, so I don’t think this counts as a “benefit” (and I think it could be confusing).

Reply:
Indeed, I removed it.